vignettes/epinow.Rmd.orig

```diff
@@ -36,10 +36,7 @@ This should be replaced with parameters relevant to the system that is being stu
 library("EpiNow2")
 options(mc.cores = 4)
 reported_cases <- example_confirmed[1:60]
-reporting_delay <- dist_spec(
-  mean = convert_to_logmean(2, 1), mean_sd = 0,
-  sd = convert_to_logsd(2, 1), sd_sd = 0, max = 10
-)
+reporting_delay <- lognormal(mean = 2, sd = 1, max = 10)
 delay <- example_incubation_period + reporting_delay
 rt_prior <- list(mean = 2, sd = 0.1)
```

Comment:
Agree in principle though this would probably require extension of the distribution interface to not automatically discretise, or only to discretise when converting to stan code (possibly a good idea anyway?).

Reply:
https://github.com/epiforecasts/EpiNow2/issues/525